bin/pulsar-client

```diff
@@ -19,7 +19,7 @@ BINDIR=$(dirname "$0")
 PULSAR_HOME=`cd $BINDIR/..;pwd`
 
 DEFAULT_CLIENT_CONF=$PULSAR_HOME/conf/client.conf
-DEFAULT_LOG_CONF=$PULSAR_HOME/conf/log4j.properties
+DEFAULT_LOG_CONF=$PULSAR_HOME/conf/log4j.shell.properties
```

Comment:
What I meant is if someone modified $PULSAR_HOME/conf/log4j.properties to change category or location of log file, they have to make similar change to $PULSAR_HOME/conf/log4j.shell.properties to get the same behavior. Might not be a big concern since it's only for the tool.